config/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "monitoringCluster": "mad",
+  "logDirectory": "logs",
+  "pipelinesDirectory": "/opt/mad/config/pipelines",
```

Comment:
I would like to see this omitted by making it optional in the configuration and then stop the directory scanning altogether when not specified. The same strategy will be used in CAgg for disabling the cluster pipeline once we have a reaggregatable datastore.